api/remoterelations/remoterelations_test.go

```diff
@@ -93,8 +94,151 @@ func (s *remoteRelationsSuite) TestWatchLocalRelationUnits(c *gc.C) {
 		callCount++
 		return nil
 	})
-	st := remoterelations.NewClient(apiCaller)
-	_, err := st.WatchLocalRelationUnits("relation-wordpress:db mysql:db")
+	client := remoterelations.NewClient(apiCaller)
+	_, err := client.WatchLocalRelationUnits("relation-wordpress:db mysql:db")
+	c.Check(err, gc.ErrorMatches, "FAIL")
+	c.Check(callCount, gc.Equals, 1)
+}
+
```

Comment:
Tedious I know, but I don't see tests for the wrong number of return values, nor for the API call itself returning an error, for any of the new calls.